mediaqueries-4/Overview.bs

```diff
@@ -1695,6 +1695,151 @@ Color Display Quality: the 'color-gamut' feature</h3>
 	you can use this feature in a negated boolean-context fashion:
 	''not (color-gamut)''.
 
+<h3 id="dynamic-range">
+Dynamic Range: the 'dynamic-range' feature</h3>
+
+	<pre class='descdef mq'>
+	Name: dynamic-range
+	Value: standard | high
+	For: @media
+	Type: discrete
+	</pre>
+</h3>
+
+High dynamic range (HDR) represents the combination of brightness, color depth, transfer function, and color gamut.
+
+<dl dfn-type=value dfn-for="dynamic-range">
+	<dt><dfn>high</dfn>
+	<dd>
+		If the following are true for the UA and the output device:
+		* color gamut is <a>p3</a>, <a>rec2020</a>
+		* color depth is greater than 24 bit
+		* has a high contrast ratio
```

Comment:
is contrast ratio defined?

Reply:
Hmmmm.... as it relates to displays I'm not sure. @vi-dot-cpp @chcunningham @mwatson2 do you all happen to know if this is define anywhere? If not, we'll need to define what "has a high contrast ratio" means